docs/chat.md

```diff
@@ -147,6 +147,9 @@ Base endpoint is: `/ocs/v2.php/apps/spreed/api/v1`
 * `call_ended` - Call with {user1}, {user2}, {user3}, {user4} and {user5} (Duration 30:23)
 * `read_only_off` - {actor} unlocked the conversation
 * `read_only` - {actor} locked the conversation
+* `listable_participants` - {actor} made the conversation listable only for participants
```

Comment:
This makes no sense 🙈 i would call it not listable if that is the current state

Reply:
here with "participant" is implied that the room is only listable by people who already joined them

but I'm fine changing it to "listable_none"